rocky/reports/report_types/definitions.py

```diff
@@ -168,6 +168,11 @@ class AggregateReportSubReports(TypedDict):
     optional: list[type[Report]]
 
 
+class SubReportPlugins(TypedDict):
+    required: list[type[Report]]
+    optional: list[type[Report]]
```

Comment:
Why is this one added? It's a duplicate of the above type and it's not even used. I suggest removing it or rename the `AggregateReportSubReports` type to a more generic name